packages/server/lib/errors.js

```diff
@@ -987,6 +987,16 @@ const getMsgByType = function (type, arg1 = {}, arg2, arg3) {
 
         https://on.cypress.io/component-testing
         `
+
+    case 'CONFIG_FILES_LANGUAGE_CONFLICT':
+      return stripIndent`
+          We've found the following Cypress config files: \`${arg2}\`, \`${arg3}\` at the locations below:
+
+          ${arg1}
+
+          JSON configuration files will be deprecated in a future release. Please fully migrate to a cypress.config.js file and re-run your command.
```

Comment:
Hmm. I reviewed these already. I used plural because there are multiple kinds of JSON configuration files, and they're all being deprecated in future releases.